src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -1,164 +1,161 @@
-        <>
-          {isApproved &&
-            !['Complete', 'Scheduled'].includes(currentStatus.overallStage) && (
-              <Tag>Approved</Tag>
-            )}
-          <StatusBlock
-            color={statusColor}
-            text={
-              currentStatus
-                ? (currentStatus.overallStage === 'Complete' &&
-                    isApproved &&
-                    'Published') ||
-                  currentStatus.overallStage
-                : 'Waiting to be scheduled...'
-            }
-            id={
-              currentStatus
-                ? `release-process-status-${currentStatus.overallStage}`
-                : 'release-process-status-WaitingToBeScheduled'
-            }
-          />
-        </>
-      )}
-
-      {currentStatus &&
-        !['Validating', 'Scheduled', 'Invalid'].includes(
+    <Details
+      className="govuk-!-margin-bottom-0 govuk-!-margin-top-1"
+      summary="View stages"
+    >
+      {checklistStyle &&
+        ![...notStartedStatuses, 'Scheduled'].includes(
           currentStatus.overallStage,
-        ) &&
-        exclude !== 'details' && (
-          <Details
-            className="govuk-!-margin-bottom-0 govuk-!-margin-top-1"
-            summary="View stages"
-          >
-            <ul className="govuk-list">
-              {Object.entries(currentStatus).map(([key, val]) => {
-                if (['overallStage', 'releaseId', 'lastUpdated'].includes(key))
-                  return null;
-                const { color, text } = statusDetailColor(val);
+        ) && (
+          <p className="govuk-!-font-weight-bold">Release process started</p>
+        )}
+      <ul className="govuk-list">
+        {Object.entries(currentStatus).map(([key, val]) => {
+          if (['overallStage', 'releaseId', 'lastUpdated'].includes(key)) {
+            return null;
+          }
+          const { color, text } = getStatusDetail(val);
+
+          if (!color) {
+            return null;
+          }
 
-                if (!color) {
-                  return null;
+          return (
+            <li key={key}>
+              <StatusBlock
+                checklistStyle={checklistStyle}
+                color={color}
+                text={
+                  checklistStyle
+                    ? `${key.replace('Stage', '')} ${text}`
+                    : `${key.replace('Stage', '')} - ${text}`
                 }
+              />
+            </li>
+          );
+        })}
+      </ul>
+    </Details>
+  );
+};
 
-                return (
-                  <li key={key}>
-                    <StatusBlock
-                      color={color}
-                      text={`${key.replace('Stage', '')} - ${text}`}
-                    />
-                  </li>
-                );
-              })}
-            </ul>
-          </Details>
-        )}
+interface ReleaseServiceStatusProps {
+  isApproved?: boolean;
+  releaseId: string;
+}
+
+const ReleaseServiceStatus = ({
```

Comment:
This name doesn't actually make a lot of sense when I think about it. Would suggest renaming it to `ReleasePublishingStatus`